### PR TITLE
Latest updates

### PR DESCRIPTION
**Pull request by ramil350**
- Set up repo
- Add main page with domain list and stats
- Scan for ssl certificates
- Deliver notifications

### CODE REVIEW

**Comment by codelion** [comment]
@RamilGilmanov  are all the files in the `/dummy` folder required?

**Comment by ramil350** [comment]
@codelion Files in spec/dummy are required for tests to run properly.

File: spec/dummy/config/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: f922981058ca96df946805a74f3a32a95749f5d1506829ffdcdb3c8a8a817be1e4f0ef77fe60e7c3435e22c493172e15abf380be0c856502b07bce5689fc8d40
```
Comment (codelion) [inline spec/dummy/config/secrets.yml:14]: Do we hard code these for dev and test environment in other tools as well?

Comment (ramil350) [inline spec/dummy/config/secrets.yml:14]: This secret has nothing to do with real application. This is for testing only.

**Comment by codelion** [comment]
Ok, GTM!

**Pull request merged**